api/src/controllers/login.js

```diff
@@ -83,6 +90,7 @@ const renderLogin = (req, branding) => {
     .then(([ template, locales ]) => {
       return template({
         branding: branding,
+        browserLocale: getBestLocaleCode(req.headers['accept-language'], locales),
```

Comment:
I wonder if this should be merged with `defaultLocale`? There's no point having both in the page if we're only going to use one.

Reply:
Got it.